lib/membrane_h264_plugin/parser.ex

```diff
@@ -31,24 +32,47 @@ defmodule Membrane.H264.Parser do
   * in the `:au_aligned` mode, the output buffers have their `:pts` and `:dts` set to `:pts` and `:dts` of the input buffer
   (holding the whole access unit being output)
 
+  The parser also allows for conversion between stream structures. The available structures are:
+  * Annex B, `:annexb` - In a stream with this structure each NAL unit is prefixed by three or
+  four-byte start code (`0x(00)000001`) that allows to identify boundaries between them.
+  * avc1, `:avc1` - In such stream a DCR (Decoder Configuration Record) is included as metadata
```

Comment:
I wouldn't say that it is included as `metadata` (in context of membrane it might be easily confused with `metadata` field in `Membrane.Buffer`). Let's just say that it is included in stream_format